app/src/main/java/org/mozilla/fenix/utils/RunIfFragmentAttached.kt

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.utils
+
+import androidx.fragment.app.Fragment
+
+/**
+ * Run the [block] only if the [Fragment] is attached.
+ *
+ * @param block A callback to be executed if the container [Fragment] is attached.
+ */
+internal inline fun Fragment.runIfFragmentIsAttached(block: () -> Unit) {
```

Comment:
I know I said utils but now that I'm looking at it, maybe we should put this in the already existing `ext/Fragment.kt` file. What do you think?

Reply:
More logical 👍